site/utils/apiUtils/index.ts

```diff
@@ -37,3 +37,12 @@ export const redirectToError = (
     logger.error(message, { context, error: error.stack });
     redirectTo(res, "/error");
 };
+
+export const isValueInArray = (value: unknown, array: string[]): boolean => {
```

Comment:
I dont really understand why we need a wrapper function around 'includes' - if we have a value, and an array, we can already do

`[].includes(value)` and that will give us a boolean. 

We can remove this function.

Reply:
wrapper function removed